index.js

```diff
@@ -2,6 +2,7 @@
 // HMAC Duo v1 API Signing Pre-Request script.
 //
 // - Tory Berra 9/19/2018
+var CryptoJS = require("crypto-js");
```

Comment:
This is unused.

Reply:
Never mind, I was using it without the reference. Maybe I need a package.json eventually.